docker/Dockerfile

```diff
@@ -24,11 +22,33 @@ ENV NODE_OPTIONS '--max-old-space-size=32768'
 # Use this for faster iterations on nginx config
 # RUN mkdir -p /app/dist  && echo hello > /app/dist/index.html
 
+# Copying files more specifically lets us reuse the build cache and iterate more quickly
+# COPY . .
+# RUN corepack enable \
+#   && yarn install \
+#   && yarn format:check \
+#   && yarn lint:check \
+#   # && yarn test \
+#   && yarn build
+
+RUN mkdir -p public src scripts .husky .yarn .storybook cypress \
+  && rm -rf node_modules
+COPY public/ public
+COPY scripts/ scripts
+COPY .yarn/ .yarn
+COPY .husky/ .husky
+COPY .storybook/ .storybook
+COPY .gitignore .yarnrc .prettierignore .prettierrc.json .nvmrc .eslintrc.js .env.development .env.production babel.config.js cypress.config.ts jest.config.ts package.json tsconfig.json vite.config.js yarn.lock index.html graphql.schema.json codegen.yml .
+
 RUN corepack enable \
- && yarn install --offline \
- && yarn format:check \
+ && yarn cache clean \
+ && yarn install
```

Comment:
I would like to keep this installing from the offline mirror so that we don't run into issues with the yarn package registry being unreliable